exchanges/graphcache/src/default-storage/index.ts

```diff
@@ -0,0 +1,167 @@
+import { stringifyVariables } from '@urql/core';
+import { SerializedEntries, SerializedRequest, StorageAdapter } from '../types';
+
+export interface StorageOptions {
+  idbName?: string;
+  maxAge?: number;
+}
+
+export const makeDefaultStorage = (opts?: StorageOptions): StorageAdapter => {
+  if (!opts) opts = {};
+
+  const DB_NAME = opts.idbName || 'graphcache-v3';
+  const ENTRIES_STORE_NAME = 'entries';
+  const METADATA_STORE_NAME = 'metadata';
+
+  const batch: Record<string, string | undefined> = Object.create(null);
+  const timestamp = Math.floor(new Date().valueOf() / (1000 * 60 * 60 * 24));
+  const maxAge = timestamp - (opts.maxAge || 7);
+
+  const database$ = new Promise<IDBDatabase>((resolve, reject) => {
+    const req = indexedDB.open(DB_NAME, 1);
+
```

Comment:
Do we want to watch for `req.onblocked` here as well?

Reply:
I'm not sure how that works exactly 🤔  do you want to push a change for that?